portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,86 @@
-#content {
-  margin-left: auto;
-  margin-right: auto;
-  width: 650px;
+:root {
+  --primary: #0c4682;
+  --spacing-lg: 2rem;
+  --spacing-md: 1rem;
+  --spacing-sm: 0.5rem;
+  --spacing-xs: 0.25rem;
+  --border-style: thin black solid;
+  --max-body-width: 1080px;
+
+  font-family: 'Open Sans', sans-serif;
+}
+
+html, body {
+  margin: 0;
+  padding: 0;
+}
+
+body {
+  max-width: var(--max-body-width);
+  margin: auto;
+}
+
+h1, h2, h3 {
+  font-weight: 600;
+}
+
+header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: var(--spacing-md) var(--spacing-lg);
+  flex-wrap: wrap;
+}
+
+header > .contact-container {
+  display: grid;
+  grid-template-columns: auto 1fr;
+  grid-auto-rows: min-content;
+  gap: var(--spacing-sm);
```

Comment:
nit: gap doesn't have great browser compatibility. consider using something else (flexbox is a more modern used alternative)

https://caniuse.com/#search=gap

Reply:
Using gap in an element with display: flex is not widely supported. In this case though, it is being used with display: grid, which is supported by +90% browsers (shows up second on the link you sent)